sdk/security/keyvault/azcertificates/build.go

```diff
@@ -1,4 +1,4 @@
-//go:generate tsp-client update
+//go:generate tsp-client update --local-spec-repo /home/grace/code/azure-rest-api-specs/specification/keyvault/Security.KeyVault.Certificates
```

Comment:
👀

Reply:
fixed in [edc5b12](https://github.com/Azure/azure-sdk-for-go/pull/24199/commits/edc5b125479c42864ce611d2208c816664fdd250)